src/utils/time.ts

```diff
@@ -29,18 +21,16 @@ export const toMoment = (value?: moment.MomentInput): moment.Moment => {
  * @param  {String} date   the date to calculate number of days since
  * @return {Number} an integer of the number of days
  */
-export const daysSince = (date: string): number => {
-    const diff = toMoment().startOf('day').diff(toMoment(date).startOf('day'), 'days');
-    return !date ? 0 : diff;
-};
+export const daysSince = (date: string): number =>
+    toMoment().startOf('day').diff(toMoment(date).startOf('day'), 'days');
```

Comment:
it was a bit odd that we check for !date, which wouldn't be an issue since date prop is already type safe, and we should pass in the correct type here. And if the return is wrong, its most likely we did not pass in the correct value to this function.

Also did a sanity check on this, its only being used in AdvertiserNameStats, and the values are correct there